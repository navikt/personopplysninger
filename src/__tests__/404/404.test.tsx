import { render } from '@testing-library/react';
import PageNotFound from '@/pages/404/404';

describe('PageNotFound', () => {
    it('renders correctly', () => {
        const { asFragment } = render(<PageNotFound />);
        expect(asFragment()).toMatchSnapshot();
    });
});
