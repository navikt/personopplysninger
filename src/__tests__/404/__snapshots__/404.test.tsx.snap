// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound renders correctly 1`] = `
<DocumentFragment>
  <div
    class="notfound__container"
  >
    <h1
      class="navds-heading navds-heading--large"
    >
      404
    </h1>
  </div>
</DocumentFragment>
`;
